INN/umbrella-currentorg
# 122: Filtering of project search

## Changes

This pull request makes the following changes:

- adds filters for project organization type and project category
- adds first-pass styles for the project terms, being the two mentioned above and the project status. Different colors are applied to each to aid in debugging.
- adds first-pass styles for the search controls on desktop, to make them appear in a single line and to make the control for the project categories into a dropdown

![Screen Shot 2020-05-28 at 14 32 35](https://user-images.githubusercontent.com/1754187/83180639-b99f2a80-a0f1-11ea-9f80-07195e150b1d.png)
![Screen Shot 2020-05-28 at 14 32 41](https://user-images.githubusercontent.com/1754187/83180641-ba37c100-a0f1-11ea-9463-544bc233fe20.png)

## Why



For #

## Testing/Questions

Features that this PR affects:

- search on `/local-that-works/` and on any page with the project shortcode
- filtering of that search
- LMP on that search


Questions that need to be answered before merging:

- [ ] What should the actual styles for this look like, on desktop and on mobile?
- [ ] Do we want to try to keep this hack of a dropdown, improve it, or replace it with a scrollable box of limited height containing the list of terms?
- [ ] The term links on the individual project items are links to those term archives, not to this page's search results. Do we want to make that the case?
- [ ] The search params for project org type and project category use numeric term IDs instead of slugs. Should they use slugs instead?

Steps to test this PR:

1. search for things. Filter by categories and other things. Click the LMP button.
